nixos/modules/services/networking/firewall.nix

```diff
@@ -53,6 +65,16 @@ let
       }
     '';
 
+  getTargetForPolicy = policy:
+    if policy == "ACCEPT"
+    then "nixos-fw-accept"
+    else "nixos-fw-refuse";
```

Comment:
This seems a bit odd (without looking at the rest). If I hear policy I'd expect the following:

> Only built-in (non-user-defined) chains can have policies, and neither built-in nor user-defined chains can be policy targets.